### DISCUSSION TITLE
2.7.2

### DISCUSSION THREAD

**Discussion by matidau** [Z-Hub/Z-Push#58] [Announcements]
## What's Changed
* Fix some PHP8 errors for IMAP backend by @DoobleD in https://github.com/Z-Hub/Z-Push/pull/31
* corrected (dead-)links by @PhilPhonic in https://github.com/Z-Hub/Z-Push/pull/37

## New Contributors
* @PhilPhonic made their first contribution in https://github.com/Z-Hub/Z-Push/pull/37

**Full Changelog**: https://github.com/Z-Hub/Z-Push/compare/2.7.1...2.7.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Z-Hub/Z-Push/releases/tag/2.7.2'>2.7.2</a>.</em>

**Comment by marclaporte** [comment]
How can it also be published here? https://z-push.org/category/news/index.html

**Reply by matidau** [reply]
I'm updating the website, changing some of the scraped code into include templates. After that is done I'll update the website. Previous updates have been very manual and error prone.

Cheers,
Mat

**Reply by matidau** [reply]
Updated website
https://z-push.org/z-push-2-7-2-released/
